features/filiatie.feature

```diff
@@ -0,0 +1,618 @@
+    
+    Er zijn twee soorten kadastraal onroerende zaken, een perceel en een appartementsrecht. Een perceel is een begrensd deel van het Nederlands grondgebied. Een appartementsrecht omvat de bevoegdheid tot het gebruik van bepaalde gedeelten van een gebouw die blijkens hun inrichting bestemd zijn of worden om als afzonderlijk geheel te worden gebruikt. 
+    
+    De filiatie van een perceel wordt op een andere manier geregistreerd en daardoor ook op een andere manier in de API getoond dan een appartementsrecht.
+
+    Een perceel kan worden gesplitst in meerdere nieuwe percelen. Een perceel kan ook worden samengevoegd met één of meerdere andere percelen. 
+    Wanneer een perceel wordt gesplitst of samengevoegd vervallen de oude kadastraal onroerende zaken en ontstaan er nieuwe kadastraal onroerende zaken. De vervallen kadastraal onroerende zaken (percelen) zijn dan nog wel raadpleegbaar.
+
+    Een perceel kan worden gesplitst in meerdere appartementsrechten. Zowel het perceel als de daaruit gesplitste appartementsrechten zijn kadastraal onroerende zaken. Bij splitsing van een perceel in appartementsrechten blijft de kadastraal onroerende zaak van het "grondperceel" actueel, het is dan niet vervallen.
+
+    Een appartementsrecht kan worden gesplitst in meerdere nieuwe appartementsrechten. Dit heet "ondersplitsing" van het appartementsrecht. In dat geval wordt de oude kadastraal onroerende zaak (het oude appartementsrecht) "sluimerend". Dat oude appartementsrecht is dan dus niet vervallen en blijft ook raadpleegbaar.
+
+    Wanneer een appartementsrechtsplitsing moet worden gewijzigd, bijvoorbeeld wanneer appartementsrechten worden samengevoegd, dan vervallen de oude kadastraal onroerende zaken (appartementsrechten). De oude kadastraal onroerende zaken (appartementsrechten) vervallen dan en zijn niet meer raadpleegbaar. De filiatie van de oude kadastraal onroerende zaken (vervallen appartementsrechten) wordt dan overgenomen in de nieuwe kadastraal onroerende zaken (appartementsrechten).
+
+    In een overzicht:
+    | gebeurtenis                               | status oude kadastraal onroerende zaak/zaken | oude kadastraal onroerende zaak/zaken raadpleegbaar |
+    | splitsen van perceel in nieuwe percelen   | vervallen                                    | ja                                                  |
+    | samenvoegen van percelen                  | vervallen                                    | ja                                                  |
+    | splitsen perceel in appartementsrechten   | actueel                                      | ja                                                  |
+    | ondersplitsing van appartementsrecht      | sluimerend                                   | ja                                                  |
+    | wijzigingssplitsing van appartementsrecht | vervallen                                    | nee                                                 |
+
```

Comment:
Ik vraag mij af of dit overzicht niet beter in Gherkin kan worden geschreven. Nu doe ik zelf een vertaling wat hier staat. Hier wordt status oude KOZ genoemd, maar ik kom geen status property tegen bij KadastraalOnroerendeZaken schema component. Dus zoiets:

Gegeven perceel P1 is gesplitst in nieuwe percelen
Als "/kadastraalonroerendezaken/P1" wordt gevraagd
Dan bevat het antwoord de status code 200
En bevat het antwoord "indicatieVervallen": true

Gegeven percelen P2 en P3 zijn samengevoegd
Als "/kadastraalonroerendezaken/P2" wordt gevraagd
Dan bevat het antwoord de status code 200
En bevat het antwoord "indicatieVervallen": true

Gegeven perceel P4 is gesplitst in appartementsrechten
Als "/kadastraalonroerendezaken/P4" wordt gevraagd
Dan bevat het antwoord de status code 200
En bevat het antwoord geen property "indicatieVervallen"

Gegeven appartementsrecht A1 is ondergesplitst
Als "/kadastraalonroerendezaken/P4" wordt gevraagd
Dan bevat het antwoord de status code 200
En bevat het antwoord "indicatieSluimerend": true

Reply:
ik maak eerst een tabel met overzicht, daarna worden al deze situaties getoond in Gherkin scenario's. Bijvoorbeeld de eerste die je noemt is het scenario op regel 36. De tweede situatie is scenario op regel 95, enz.